src/Sylius/Behat/Context/Api/Shop/CustomerContext.php

```diff
@@ -413,7 +413,7 @@ public function iShouldBeNotifiedThatItHasBeenSuccessfullyChanged(): void
      */
     public function iShouldBeNotifiedThatProvidedPasswordIsDifferentThanTheCurrentOne(): void
     {
-        Assert::same($this->customerClient->getLastResponse()->getStatusCode(), 400);
+        Assert::same($this->customerClient->getLastResponse()->getStatusCode(), 422);
```

Comment:
Why some assertions are rewritten from expecting 400 to 422 and some other ones (like the one below in this file) are changed from expecting 400 to either 400 or 422?

Reply:
@pamil @lchrusciel  @GSadee This code was 422 but was wrong showed in docs, this PR fixed it in api-platform 2.6: https://github.com/api-platform/core/pull/4086